DuckDuckGo/NetworkProtectionRootView.swift

```diff
@@ -24,20 +24,20 @@ import Core
 
 struct NetworkProtectionRootView: View {
 
-    let statusViewModel: NetworkProtectionStatusViewModel
-
-    init() {
+    @StateObject
```

Comment:
Should've been a `@StateObject` all along since this was created in the root view.

Looking even beyond, this should probably be instantiated elsewhere but that's far beyond the scope of this PR.